<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;C:/Users/huynh/OneDrive/Desktop/VMmanager/Python_Water-Meter_Detected/GUI/loginBG.jpg&quot;);
background-repeat: no-repeat;
background-position: center;
</string>
   </property>
   <widget class="QLineEdit" name="UserEmail">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: lightgray; /* Change this to any color */
    color: black; /* Text color */
    border: 1px solid black; /* Optional: Border styling */
    border-radius:10;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserPassword">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>302</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: lightgray; /* Change this to any color */
    color: black; /* Text color */
    border: 1px solid black; /* Optional: Border styling */
    border-radius:10;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Mật khẩu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>371</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 242, 166);</string>
    </property>
    <property name="text">
     <string>Đăng nhập</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Đăng nhập</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-5</y>
      <width>1321</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../image/homeBG.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>UserPassword</zorder>
   <zorder>btnLogin</zorder>
   <zorder>label_3</zorder>
   <zorder>UserEmail</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1310</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>UserEmail</sender>
   <signal>returnPressed()</signal>
   <receiver>UserPassword</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UserPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
